.travis.yml

```diff
@@ -1,17 +1,14 @@
 language: ruby
+sudo: false
+cache: bundler
+bundler_args: '--without production development'
 
 rvm:
   - 2.2.2
 
-env: DATABASE_URL=postgres://postgres@localhost/timeoverflow_test
```

Comment:
Why not the `DATABASE_URL` ? Besides making 12-factors more commonplace, in my opinion this is the biggest contribution of Heroku to the rails app concept.I would love to see the same for SMTP in fact.